lib/api/entities.rb

```diff
@@ -151,6 +151,35 @@ class Teams < Grape::Entity
       }, if: { type: :internal } { |t| t.namespaces.count }
     end
 
+    class TeamMembers < Grape::Entity
```

Comment:
I wonder if we can either subclass `Users` or somehow share as much as possible from the `Users` entity ...

Reply:
If we present the `team_user.user` instead of `team_user` and add the extra fields, I guess we could inherit from `Users` entity. I don't have a strong opinion on this one to be honest.

Reply:
Maybe it's fine for now, just wondering because they share some fields. This is something we can discuss in #1500.

Reply:
Makes sense. 👍